src/models/elan1-r1/README.md

```diff
@@ -0,0 +1,29 @@
+# Eland 1U (elan1-r1)
+
+![Eland 1U](./img/elan1-r1-frontports.avif)
+
+The System76 Eland 1U is a rack-mount server with the following specifications:
+
+- Operating System
+    - Ubuntu Server 22.04 LTS (64-bit)
+- CPU options
+    - Supports AMD EPYC™ 7003 series processor family
+    - Single processor
+- Memory
+    - Up to 4096GB (16x256GB) ECC DDR4 @ 3200 MHz
+- Networking
+    - 2x 1GbE LAN ports (1 x Intel® I350-AM2)
```

Comment:
Intel ARK can be linked here: https://ark.intel.com/content/www/us/en/ark/products/52968/intel-ethernet-controller-i350am2.html